Source/OpenTK/BlittableValueType.cs

```diff
@@ -57,7 +57,11 @@ public static class BlittableValueType<T>
         static BlittableValueType()
         {
             Type = typeof(T);
+#if _NET_CORECLR
+            if (Type.GetTypeInfo().IsValueType && !Type.GetTypeInfo().IsGenericType)
```

Comment:
Why not just override with Type.GetTypeInfo instead of adding it under the compilation flag? It's supported in dot net 4.5 as well (and the recommended way, as far as I understand). Is there a need to support .Net 4? 
Also, maybe it would be better to take Type.GetTypeInfo into a variable to avoid calling it twice inside the if.